requirements.txt

```diff
@@ -0,0 +1 @@
+boto==2.38.0
```

Comment:
Perhaps ``>=`` instead? (just thinking for when new versions of boto get released, although the boto version requirement is much lower).